content/sensu-go/6.3/sensuctl/create-manage-resources.md

```diff
@@ -145,15 +145,16 @@ The following table describes the command-specific flags.
 --------------------|---|---|---|
 `AdhocRequest` | `adhoc_request` | `Asset` | `asset`
 `CheckConfig` | `check_config` | `ClusterRole`  | `cluster_role`
-`ClusterRoleBinding`  | `cluster_role_binding` | `Entity` | [`Env`][24]
-`entity` | [`EtcdReplicators`][29] | `Event` | `event`
-`EventFilter` | `event_filter` | [`GlobalConfig`][11] | `Handler` 
-`handler` | `Hook` | `hook` | `HookConfig`
-`hook_config` | `Mutator` | `mutator` | `Namespace`
-`namespace` | `Role` | `role` | `RoleBinding`
-`role_binding` | [`Secret`][28] | `Silenced` | `silenced`
-[`User`][8] | `user` | [`VaultProvider`][24] | [`ldap`][26]
-[`ad`][25] | [`oidc`][37] | [`TessenConfig`][27] | [`PostgresConfig`][32]
+`ClusterRoleBinding`  | `cluster_role_binding` | [`Component`][38] | `Entity`
+[`Env`][24] | `entity` | [`EtcdReplicators`][29] | `Event`
+`event`| `EventFilter` | `event_filter` | [`GlobalConfig`][11]
+`Handler`  | `handler` | `Hook` | `hook`
+`HookConfig` | `hook_config` | `Mutator` | `mutator`
+`Namespace` | `namespace` | `Role` | `role`
+`RoleBinding` | `role_binding` | [`RuleTemplate`][39] | [`Secret`][28]
+`Silenced` | `silenced` | [`User`][8] | `user`
+[`VaultProvider`][24] | [`ldap`][26] | [`ad`][25] | [`oidc`][37]
+[`TessenConfig`][27] | [`PostgresConfig`][32]
```

Comment:
Linked pages are not implemented yet but will be the Components reference and Rule templates reference, respectively.